### ISSUE TITLE
Add detached load control to KeypadLinc

### ISSUE THREAD

**Issue by TD22057** [TD22057/insteon-mqtt#109]
KeypadLinc can operate in "detached load" mode where button 1 (and 8) don't control the load.  That moves the load to group 9.  From the dev guide:

Detach Load Notes:

- Detach Load is only supported through software. It cannot be turned on/off from the device. 
- When Detach Load is turned On, the load will then be moved/controlled by Group 9
  - Link to Group 9 using the SET Button.
  - Link to Group 1 by using the ON or OFF Button. 
  - Link to Buttons A-H using the A-H buttons. 
- Control of the LEDs when Detach Load is enabled
  - To control the Group 1 LEDs, simply send the Keypad a Group 1 On/Off command
  - To control the other LEDs, use the Extended 2E Set LED Bit Mask command to set those LEDs 
- Getting The Status of Groups 1-9
  - To get the Status of the 8 LEDs (Group 1-8), use the Read Data 2E command and the Data 11 byte contains the LED State information.
  - To get the Status of Group 9 (The Load), use the Standard status request command Cmd1=19 Cmd2=00

@jrevans

**Comment by TD22057** [comment]
To find the current state, use a standard direct 0x1f, 0x05 message.  If bit 5 is 1, the load is detached.  To detach the load, send an extended direct message 0x20, 0x1b.  To attach the load, send an extended direct message 0x20, 0x1a. 

One question is to figure out if the KeypadLinc device (and probably the device db meta data) needs to keep track of this flag or not.  It may be enough to just add the get/set commands for this and allow group 9 to be a valid input to the commands.  

Another question: how is the load controlled when detached?  A group 9 scene responder will control it but there may not be a direct command to do so.  If that's true, see IOLinc which uses an unused number in the modem virtual scene table and issues that command to control the device.  This would allow KeypadLinc to control group 9 if that command can't be found.

**Comment by house97** [comment]
I have a 6 button KPL controlling a fanlinc, and simply pairing the light group on the fanlinc with button 1 of the KPL.  Obviously the load line of the KPL is capped (and I assume you would do that here too).
Is detatched load control something that is specific to 8 button KPLs?  Or am I missing something that could be of value to me?

**Comment by TD22057** [comment]
Not exactly.  Detached load is when you have the load wired connected to a load you want to control but you don't want to control it from the switch itself (i.e. none of the buttons on the switch will change the load or show the load state).  This feature let's you control the load as group 9 from another insteon device.

**Comment by house97** [comment]
thank you for clarifying.  Understanding what "detatched load" means in this context makes all the difference.
